src/index.html

```diff
@@ -1,13 +1,325 @@
+              <a href="tel:+123455555555" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Your senses.
+              <br>
+              Set free.
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <img src="./images/logo.png" alt="logo">
+          </a>
+          <div class="menu__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#speakers">Speakers</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#earphones">Headphones</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#categories">Televisions</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#categories">Soundbars</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#world-of-b&o">World of B&O</a>
+              </li>
+            </ul>
+          </nav>
+          <a href="tel:+123455555555" class="menu__phone-number">
+            +1 234 5555-55-55
+          </a>
+
+          <a href="tel:+123455555555" class="menu__call-to-order">
+            Call to order
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content-first-type">
+          <section class="recommended" id="recommended">
+            <h2 class="section-title">Recommended</h2>
+            <div class="recommendet__products">
+              <article class="product recommendet__product">
+                <img
+                src="images/product/BEOSOUND A5.png"
+                alt="BEOSOUND A5"
+                class="product__photo"
+                >
+                <h3 class="product__title">BEOSOUND A5</h3>
+                <p class="product__description">
+                  Huge sound anywhere – in the home or outdoors. Beosound A5 is a Wi-Fi and Bluetooth speaker with long-lasting battery and wireless charging built in.
+                </p>
+                <p class="product__price">$ 1299.00</p>
+              </article>
+
+              <article class="product recommendet__product">
+                <img
+                src="images/product/BEOPLAY HX.png"
+                alt="BEOSOUND A5"
+                class="product__photo"
+                >
```

Comment:
Need to check and fix it everywhere
```suggestion
                <img
                  src="images/product/BEOPLAY HX.png"
                  alt="BEOSOUND A5"
                  class="product__photo"
                >
```